CSSfile.css

```diff
@@ -71,53 +70,70 @@
   margin-right: 0;
 }
 
-.logo{
-  font-family: 'Hammersmith One', sans-serif;
+.logo {
+  font-family: "Hammersmith One", sans-serif;
   font-size: 50px;
-  color:whitesmoke;
+  color: whitesmoke;
   display: block;
   text-align: center;
   width: auto;
   position: sticky;
   margin-top: 10px;
 }
 
-.yinyang{
+.yinyang {
   width: 200px;
   height: 200px;
   z-index: 999;
   position: relative;
   scale: 1.5;
 }
 
-.platform{
+.platform {
   width: 300px;
   height: 300px;
   z-index: -1;
-  scale:2;
+  scale: 2;
+  filter: drop-shadow(5px 5px 2px rgb(0, 0, 0));
```

Comment:
Crazy. I really hope you get the hacktoberfest T-Shirt. I mean this is ground breaking.

Reply:
thanks